### PR TITLE
Remove protobuf output file.

### PR DESCRIPTION
**Pull request by baldwinn860**
Shouldn't be checked in.

### CODE REVIEW

**Comment by baldwinn860** [comment]
done.

File: .gitignore [hunk 1]
```diff
@@ -13,6 +13,7 @@
 *_embed.go
 gapir.log
 gapis.log
+*.pb.go
```
Comment (pmuetschard) [inline .gitignore]: Thus far, the pattern has been to add the file by name to a .gitignore file in the same dir.

Comment (baldwinn860) [inline .gitignore]: oh that makes sense, fixed

File: gapis/api/gvr/.gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 /api.pb.go
+resolvables.pb.go
```
Comment (pmuetschard) [inline gapis/api/gvr/.gitignore]: Add the slash in the front, it means to only exclude it in the current dir rather than in it and it's children.

Comment (baldwinn860) [reply gapis/api/gvr/.gitignore]: done

**Pull request merged**